modules/hound.nix

```diff
@@ -0,0 +1,29 @@
+{ config, lib, ... }:
+
+let
+  cfg = config.services.hound;
+in
+{
+  options = {
+    services.hound = {
+      repos = lib.mkOption {
+        type = with lib.types; listOf str;
+        default = [];
+        example = [ "https://github.com/NuschtOS/nixos-modules.git" ];
+        description = ''
+          A list of repos which should be fetched from their default branch. The display name is derived using builtins.baseNameOf and .git is stripped
+        '';
+      };
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    services.hound.settings = {
```

Comment:
Maybe check if repos is not empty? Just enabling hound theoretically does not mean to enable this Module.